cryptoki/src/mechanism/elliptic_curve.rs

```diff
@@ -93,6 +93,17 @@ impl<'a> EcKdf<'a> {
         }
     }
 
+    /// The sha256 transformation as defined in the x9 standard. The
+    /// derived key is produced by concatenating hashes of the shared
+    /// value followed by 00000001, 00000002, etc. until we find
+    /// enough bytes to fill the `CKA_VALUE_LEN` of the derived key.
+    pub fn sha256_x9() -> Self {
```

Comment:
```suggestion
    pub fn sha256() -> Self {
```
I think SHA256 is actually specific enough here, since the type makes it clear it's a KDF.